cypress/integration/cancel-repair/cancel-work-order.spec.js

```diff
@@ -0,0 +1,152 @@
+/// <reference types="cypress" />
+
+import 'cypress-audit/commands'
+
+describe('Cancel work order', () => {
```

Comment:
I like this test.

One approach we could take for a readability bonus is to use a Date object we control and refer to within the test. I was reading a bit about cypress fixtures and JS dates. Looks like we might be able to show the reader exactly what date we're comparing to here and what the difference in minutes should be for the boundaries of our tests. It makes use of `.then` in cypress (https://docs.cypress.io/api/commands/fixture.html#Modifying-fixture-data-before-using-it).

This is a proposed idea in pseudocode:

```
  const workOrderRaisedTime = new Date()

  beforeEach(() => {
    //...

      cy.fixture('repairs/work-order.json')
      .then((workOrder) => {
        workOrder.dateRaised = workOrderRaisedTime.toISOString()  // override fixture property
      })
      .as('workOrder')
      // ...
  })

// ...
      
   it('Cancel a work order', () => {
        const cancelTime = new Date(workOrderRaisedTime)

        cancelTime.setMinutes(workOrderRaisedTime.getMinutes() + 59)

        cy.clock(cancelTime)

        // ... etc 
```
Might need an update for the check we do on the request object (e.g. `eventTime` we send up).

And a similar approach for the following test, except maybe using this 

` cancelTime.setMinutes(workOrderRaisedTime.getMinutes() + 61)`

Reply:
Yes, I agree this would improve readability and convey what the tests are trying to achieve a bit better here 👍